pulseaudio-control.bash

```diff
@@ -38,6 +45,31 @@ function getCurVol() {
 }
 
 
+# Saves the `device.description` property of the sink passed by parameter into a variable named `curDescr`.
+function getCurDescr() {
+  curDescr=$(pacmd list-sinks | sed -n -e '/index: '"$1"'/,/index/ p' | grep 'device.description' | sed 's/.*"\(.*\)"/\1/g')
+}
+
+
+# Saves the name of the sink passed by parameter into a variable named `curName`.
+function getCurName() {
```

Comment:
Why does `getCurName` return the `alsa_output.pci-0000_00_1f.3.analog-stereo`-like name? Shouldn't it be better to already try to obtain the nickname inside it, and using `alsa_output.pci-0000_00_1f.3.analog-stereo` in case it failed?

Reply:
well, getCurName() ist required in either case to determine the lookup-key. In that sense it's a "lower level" utility method, which is used in getCurNick(), which does exactly what you describe :)